templates/vue-demo-store/app/components/account/AccountOrderDownloads.vue

```diff
@@ -6,7 +6,7 @@ const props = defineProps<{
   documents: Schemas["Document"][];
 }>();
 
-const { getDocumentFile } = useOrderDetails(props.documents[0].orderId);
+const { getDocumentFile } = useOrderDetails(props.documents[0]?.orderId || "");
```

Comment:
`orderId` cannot be `""`. This situation would break the request and component. Props tyle should be changed to the non-empty array. Something like `[T, ...T[]]`

Reply:
covered